backend/src/main/java/gov/cdc/usds/simplereport/logging/AuditLoggingInstrumentation.java

```diff
@@ -80,7 +79,7 @@ public void onCompleted(ExecutionResult result, Throwable t) {
       LOG.trace("End of execution, audit entry being saved.");
       List<String> errorPaths =
           result.getErrors().stream()
-              .map(GraphQLError::getPath)
+              .map(e -> e.getPath() == null ? new ArrayList<Object>() : e.getPath())
```

Comment:
**question (non-blocking):** Does `Collections.emptyList()` not work here?

Reply:
It didn't... because `Collections` wasn't imported 🙂 Reverted in 37aeffc because it's a little more readable.